build.sbt

```diff
@@ -181,7 +181,8 @@ lazy val docs = project
   .settings(noPublishSettings)
   .settings(docSettings)
   .settings(commonJvmSettings)
-  .dependsOn(coreJVM, freeJVM)
+  .settings(libraryDependencies += "com.github.alexarchambault" %% "scalacheck-shapeless_1.13" % "1.1.5" % Compile)
```

Comment:
I would strongly discourage adding a third party dependency unless it is _really_ essential. cats is quite low down the compile chain so the fewer dependencies it has that could get in the way of upgrades the better. And for thing like testing new scala compilers (eg 2.13.0.M1, native,etc) it is not the dependencies that a third party library brings into cats that is the issue, rather the dependencies needed to build that 3rd party library in the first place.

So I'm not saying it should not have any at all, but that we have to be careful.

Reply:
As a starting suggestion/discussion.....

The same is true for `testkit`, that I have suggested we split into two: a `testkit` that has no dependencies other than pure cats, and another `testkit-scalatest` that adds the scalatest dependency. The idea here is that we make the the base testkit independent to the testing frameworks. But what about specs2, should we also add a `testkit-specs2`, `testkit-utest` and so on? 

And what about, say, [cats-check](https://github.com/non/cats-check). For sure we _can't_ add that as a dependency.

So maybe now is a good time to address this. Maybe we need some kind of uber-cats repo that brings these things together?

Reply:
You raise a good point, I was hesitant to add the dependency before. I can revert the last commit and make the `Arbitrary` instance explicit with a mention to scalacheck-shapeless.

Reply:
I agree we should be very cautious about adding dependencies to the project. In this particular case, it's a dependency added to the docs generation project, so no downstream impact. That's why I didn't object it. However, now I realized that it does impact the docs build in scala 2.13.

 "Making the `Arbitrary` instance explicit with a mention to scalacheck-shapeless" would make more sense to me.